client/src/components/NavBar.tsx

```diff
@@ -1,17 +1,26 @@
+import { ReactComponent as Logo } from 'assets/MHPLogo.svg';
 import React from 'react';
-import { NavLink } from 'react-router-dom';
 import { Nav, Navbar, NavDropdown } from 'react-bootstrap';
-import { useBikeVersion, bikeVersions } from 'router';
-import { ReactComponent as Logo } from 'assets/MHPLogo.svg';
+import { NavLink } from 'react-router-dom';
+import { VersionInfo } from '../router';
 import styles from './NavBar.module.css';
 
+/**
+ * @property {VersionInfo} bikeVersion Version info for bike
+ * @property {VersionInfo[]} bikeVersions Version info for all bikes
+ */
+export interface NavBarProps {
+  bikeVersion: VersionInfo | null,
+  bikeVersions: VersionInfo[]
+}
+
 /**
  * Navigation bar component
  *
- * @returns {React.Component} Component
+ * @param {NavBarProps} props Props
+ * @returns {JSX.Element} Component
```

Comment:
No need to explicitly specify types in the docs when typescript can tell us

Reply:
Done
017e912